src/widgets/InputField.js

```diff
@@ -50,11 +50,17 @@ class InputWidget extends Component {
         this.props.onChange(this.props.propName, this.state.value)
     }
 
-    render ({inputName, namespace, options}) {
+    render ({inputName, namespace, options, _id, doesNotRenderLabel}) {
         const field = this.props.adapter.context
 
         const isValid = this.props.validationError || this.props.invariantError ? false : undefined
 
+        const ariaLabels = {
+            'aria-invalid': !isValid,
+            'aria-labelledby': doesNotRenderLabel ? undefined : _id,
+            'aria-label': doesNotRenderLabel ? _id : undefined
```

Comment:
I think this should be something like this:

` 'aria-label': doesNotRenderLabel ? renderString(field.label || 'inferno-formlib--InputField', options && options.lang, 'Input Field') : undefined`

because the id doesn't say much to a user.